### ISSUE TITLE
Wrong link on popover documentation

### ISSUE THREAD

**Issue by stenci** [anvilistas/anvil-extras#119]
[This page](https://anvil-extras.readthedocs.io/en/latest/guides/modules/popover.html) of the documentation has one link to the example app and one that claims to be to the same app but it's a clone of the dependency itself, it doesn't have any example form.

The code of the dependency is included in the repository, the form is not. 

It would be nice to have the clone for the app with the example form (as advertised in the page) rather than the dependency.

**Comment by stenci** [comment]
I would change this issue to "Missing documentation on popover documentation."

I stared at that `popover` function for 15 minutes, trying to figure out how to use it. Then I realized that there was a `self` argument and tried to figure out how to add it as a method to a button. Then I found [this post](https://anvil.works/forum/t/implementing-popovers-floating-forms/3775) that says "This will add popover functionality to links and buttons" and I thought "what a way to waste 30 minutes!".

The content of that post (after some cleaning) should be included in the popover documentation page.
